crowbar_framework/app/assets/javascripts/installer.js

```diff
@@ -1,86 +1,84 @@
 $(document).ready(function() {
-  $("body.installers").each(function() {
-    function statusCheck() {
-      $.ajax({
-        type: "GET",
-        dataType: "json",
-        url: Routes.status_installer_path(),
-        success: function(data) {
-          // initially steps are empty, so we need to return
-          if (!data.steps) {
-            // indicate first step after triggering installation
-            // otherwise the UI will take 3 seconds to catch up
-            if (data.installing) {
-              $("li.pre_sanity_checks").addClass("list-group-item-success");
-              $("li.pre_sanity_checks span.fa")
-                .removeClass("fa-hourglass-o")
-                .addClass("fa-circle-o-notch fa-spin");
-            }
-            setTimeout(statusCheck, 3000);
-            return;
-          }
-          var mostRecent = $("li." + data.steps[data.steps.length-1]);
-          var mostRecentIcon = mostRecent.children();
-
-          $.each(data.steps, function (index, value) {
-            $("li." + value)
-              .removeClass("list-group-item-success")
-              .children("span")
-              .removeClass("fa-hourglass-o fa-circle-o-notch fa-spin")
-              .addClass("fa-check");
-          });
-          mostRecentIcon.addClass("fa-check");
-
-          if (data.failed) {
-            mostRecent
-              .next("li")
-              .addClass("list-group-item-danger")
-              .children("span")
-              .removeClass("fa-hourglass-o fa-circle-o-notch fa-spin")
-              .addClass("fa-times");
-          } else {
-            mostRecent
-              .next("li")
-              .addClass("list-group-item-success")
-              .children("span")
+  function statusCheck() {
+    $.ajax({
+      type: "GET",
+      dataType: "json",
+      url: Routes.status_installer_path(),
```

Comment:
Identifier 'status_installer_path' is not in camel case.<br>'Routes' is not defined.